lib/simplecov/merge_helper.rb

```diff
@@ -0,0 +1,31 @@
+module SimpleCov
+  module MergeHelper
+    class << self
+      # Merges two Coverage.result hashes
+      def merge_resultsets(a, b)
```

Comment:
Putting in on ResultMerger seems sensible. I considered it, then thought maybe it'd be nicer to keep this code self-contained. I.e., ResultMerger manages `.resultset.json`, and this module manages merging results in-memory. But it was a close call, and if you prefer moving it to ResultMerger I'll do that.